lib/framescript/content.jsm

```diff
@@ -0,0 +1,91 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+const { utils: Cu, classes: Cc, interfaces: Ci } = Components;
+const { Loader } = Cu.import('resource://gre/modules/commonjs/toolkit/loader.js', {});
```

Comment:
we should use `__URI__.replace(...)` here so that overloading continues working.